ts/components/leftpane/ActionsPanel.tsx

```diff
@@ -238,6 +240,20 @@ export const ActionsPanel = () => {
     return () => clearTimeout(timeout);
   }, []);
 
+  const globalUnreadMessageCount = useSelector(getGlobalUnreadMessageCount);
+  const unreadToShow = globalUnreadMessageCount;
```

Comment:
`unreadToShow` is not needed. Just use `globalUnreadMessageCount` directly below

Reply:
made the change